Makefile

```diff
@@ -30,3 +30,10 @@ ci-armhf-push:
 ci-armhf-build:
 	(./build.sh $(TAG)-armhf)
 
+.PHONY: test-integration
+PORT?=8083
```

Comment:
This port I think may clash with AlertManager or similar, how about using a port we know like 8080, then add a high letter such as `38080`? (just check that's a valid port first)

Reply:
yep, I'm actually not using it, but forgot to edit.